docs/developer-docs/functionality/http_requests/http_requests-how-it-works.md

```diff
@@ -0,0 +1,152 @@
+# Technology Overview -- How Canister HTTP Requests Work
+
+We next give some more details on how canister HTTP requests work and important aspects to consider when using the API. We already want to note here that there are some limitations compared to regular (Web 2.0) computer programs making HTTP outcalls and considerations for programmers for successfully using this feature. Engineers who intend to use this feature are advised to read through this documentation to get up to speed quickly w.r.t. the feature.
+
+## Technology
+
+The feature allows canisters to make outgoing HTTP calls to conventional Web 2.0 HTTP servers. The response of the request can be safely used in computations, without the risk of state divergence between the replicas of the subnet. This is crucial, as it has required us to overcome some challenges which have so far prevented blockchains from implementing something like this feature.
```

Comment:
```suggestion
The HTTP requests feature allows canisters to make outgoing HTTP calls to conventional Web 2.0 HTTP servers. The response of the request can be safely used in computations, without the risk of state divergence between the replicas of the subnet.
```
(I'm being flippant here) Noone cares about the challenges _we_ had to go through, and the last sentence is pure fluff. People at most care about what problems are solved by the implementation, and actually care about how they have to work around/with implementation details.